api_tests/comments/views/test_comment_report_list.py

```diff
@@ -86,11 +86,10 @@ def test_public_node_reported_user_does_not_see_report(self):
         assert_equal(len(report_json), 0)
         assert_not_in(self.contributor._id, report_ids)
 
-    def test_public_node_non_contributor_does_not_see_report(self):
+    def test_public_node_non_contributor_does_see_report(self):
         self._set_up_public_project_comment_reports()
         res = self.app.get(self.public_url, auth=self.non_contributor.auth, expect_errors=True)
-        assert_equal(res.status_code, 403)
-        assert_equal(res.json['errors'][0]['detail'], 'You do not have permission to perform this action.')
+        assert_equal(res.status_code, 200)
```

Comment:
Non-contributors should be able to access this endpoint, but they should not be able to see any reports that are not theirs. Can you assert that no report info is returned? In other tests, I've been doing this with
``` 
report_json = res.json['data']
report_ids = [report['id'] for report in report_json]  
assert_equal(len(report_json), 0)
```